From 9c5d0a96e6467155aa6f4972ef13b13ad3e0b806 Mon Sep 17 00:00:00 2001
From: Wang Hai <wanghai38@huawei.com>
Date: Sat, 13 Aug 2022 10:52:07 +0800
Subject: [PATCH 4577/4585] [Huawei] sched/fair: Fix kabi broken in cfs_rq

Offering: HULK
hulk inclusion
category: bugfix
bugzilla: https://gitee.com/openeuler/kernel/issues/I5JGN1

--------------------------------

In struct cfs_rq, the name of 'throttled_clock_pelt' and
'throttled_clock_pelt_time' changed causing kabi broken,
use KABI_REPLACE to fix it.

Signed-off-by: Wang Hai <wanghai38@huawei.com>
---
 kernel/sched/sched.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/kernel/sched/sched.h b/kernel/sched/sched.h
index 9ef82c302e46..5a60bf6aa47d 100644
--- a/kernel/sched/sched.h
+++ b/kernel/sched/sched.h
@@ -625,8 +625,8 @@ struct cfs_rq {
 	s64			runtime_remaining;
 
 	u64			throttled_clock;
-	u64			throttled_clock_pelt;
-	u64			throttled_clock_pelt_time;
+	KABI_REPLACE(u64 throttled_clock_task, u64 throttled_clock_pelt)
+	KABI_REPLACE(u64 throttled_clock_task_time, u64 throttled_clock_pelt_time)
 	int			throttled;
 	int			throttle_count;
 	struct list_head	throttled_list;
-- 
2.17.1

